packages/paste-website/src/pages/patterns/data-export/index.mdx

```diff
@@ -0,0 +1,264 @@
+
+## Variations
+
+### Immediate export
+
+An immediate export happens when a customer can download their export on
+the same page where they clicked the export Button or Menu item.
+
+When using a Button, while an export is in progress, show the Button loading
+state. When it’s in a loading state, it will disable by default.
+
+<LivePreview
+  scope={{MenuButton, MenuItem, Menu, useMenuState, Button, ChevronDownIcon, ExportIcon}}
+  noInline
+  language="jsx"
+>
+  {exportMenu}
+</LivePreview>
+
+Most often, selecting the export Button or Menu item should trigger a “save as”
+dialog in the browser. Customers will be able to track the export’s download
+status through their browser.
+
+However in cases where you might offer more customization of the data, you
+can trigger a Modal first. In the Modal, customers can select what columns
+to export, additional file formats, etc.
+
+<LivePreview
+  scope={{
+    useUIDSeed,
+    CheckboxGroup,
+    Checkbox,
+    Modal,
+    ModalHeader,
+    ModalHeading,
+    ModalBody,
+    ModalFooter,
+    ModalFooterActions,
+    Tooltip,
+    Button,
+    ExportIcon,
+  }}
+  noInline
+  language="jsx"
+>
+  {exportModal}
+</LivePreview>
+
+### Delayed export
+
+A delayed export happens when a customer needs to wait for data to be ready.
+Use the same components as the immediate export pattern, but make sure to:
+
+1. Confirm the export was initiated by using a Toast component. Inform the customer that they’ll receive an email notification when it’s ready.
+2. Send an email notification to the customer when the export is ready, with a link to or attachment of the file(s). In this second step, use the word “Download” (example: “Download CSV”) to trigger the actual file download since the customer already clicked an export action to start the export process.
+
+<LivePreview
+  scope={{
+    Toaster,
+    useToaster,
+    Button,
+    ExportIcon,
+  }}
+  noInline
+  language="jsx"
+>
+  {exportDelayed}
+</LivePreview>
+
+## Positioning the export action
+
+Place the export Button above and right-aligned to the data it’s exporting,
+usually a data table or graph.
+
+![table with a data export button placed at the top right](../../../assets/images/patterns/data-export-example-6a.png)
+
+![chart with a data export button placed at the top right](../../../assets/images/patterns/data-export-example-6b.png)
+
+It can also be placed in a card component that shows the specific data that will
+be exported, such as a billing invoice.
+
+![A card with a data export button](../../../assets/images/patterns/data-export-example-7.png)
+
+## Starter kits
+
+### Design Templates
```

Comment:
```suggestion
```

Reply:
I feel like we could just link to Figma and CodeSandbox under a single "Starter kits" header.